noir-projects/noir-contracts/contracts/parent_contract/src/main.nr

```diff
@@ -257,7 +257,7 @@ contract Parent {
         let owner = env.create_account();
 
         // Deploy child contract
-        let child_contract = env.deploy("@aztec/noir-contracts.js/Child").without_initializer();
+        let child_contract = env.deploy("./@child_contract", "Child").without_initializer();
```

Comment:
This is incredible

Reply:
Do we need the `./`? It's an extremely minor thing, but perhaps we can just assume that its local?

Reply:
I see your point but I'd rather keep it for the time being, since this syntax is explicitly stating "access this workspace, then the package". I think we will sooner rather than later do URL imports so the visual distinction will be good.